.travis.yml

```diff
@@ -2,9 +2,6 @@ language: objective-c
 matrix:
   fast_finish: true
 env:
-  - FORMULA=zeroc-ice34 BREW_OPTS=
-  - FORMULA=zeroc-ice34 BREW_OPTS=--with-python
-  - FORMULA=zeroc-ice34 BREW_OPTS=--with-java
   - FORMULA=zeroc-ice35 BREW_OPTS=--with-python
```

Comment:
@rleigh-dundee: we haven't defined a `--with-java` for the `zeroc-ice35` formula.